packages/cra-template/template/.editorconfig

```diff
@@ -0,0 +1,15 @@
+# https://github.com/facebook/react/blob/main/.editorconfig
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
+indent_style = space
+insert_final_newline = true
+max_line_length = 80
+trim_trailing_whitespace = true
+
+[*.md]
+max_line_length = 0
+trim_trailing_whitespace = false
```

Comment:
insert_final_newline. And I think it'd be good to add this, too!
And I think false would be appropriate for insert_final_newline.
This is not a code.
It's just a document!